app/views/layouts/application.html.erb

```diff
@@ -17,7 +17,6 @@
 
     <%= yield %>
 
-     <p class="notice"><%= notice %></p>
      <p class="alert"><%= alert %></p>
```

Comment:
We don't need the alert either, as it's already included in the `_flashes.html.erb` partial